docs/guidelines/content-guidelines/03-formatting.md

```diff
@@ -36,6 +36,7 @@ Metadata names            | When you create a Markdown document, define its `tit
 Flags                     | Add the `--tls` flag to every Helm command you run.
 GraphQL queries and mutations   | The `requestOneTimeTokenForApplication` mutation calls the internal GraphQL API to get a one-time token. |
 Rights                    | The preset gives multiple resource cleaners the `list` and `delete` rights in GCP.
+Custom resources (if you want to refer to the code specifically) | In the exposed `APIRule`, use the following Kyma Gateway: `kyma-gateway.kyma-system.svc.cluster.local`. |
```

Comment:
I wonder if this example sentence should not be changed to a different one, as this one doesn't really show the difference between when you want to refer to a CR as code. Here, it could be either way, really, which makes it ambiguous to me. I'd rather use something like: ```Define this parameter in the `APIRule` metadata``` or something... What do you think?

Reply:
It could be ambiguous. I modified it, pls have a look :)